java/src/apps/GuiLafConfigPane.java

```diff
@@ -212,22 +184,29 @@ private int getDefaultFontSize() {
     static ActionListener listener;
 
     public void doFontSize(JPanel panel) {
-
+        GuiLafPreferencesManager manager = InstanceManager.getDefault(GuiLafPreferencesManager.class);
```

Comment:
@rhwood
Thanks. I've done that. I gather this is important because of its use in an Anonymous Inner Class within a public method, is that a correct interpretation?

Should I go ahead and use the private manager() method in all the other methods in java/src/apps/GuiLafConfigPane.java that use InstanceManager.getDefault(GuiLafPreferencesManager.class) as well? Some of them are public but not in anonymous inner classes. May not be necessary, but would tidy up.

Also should I do the same in java/src/apps/GuiLafConfigPaneXml.java? It would tidy up, even if not necessary. Or is it necessary there too?

Or what about java/src/apps/gui3/FirstTimeStartUpWizard.java or java/src/jmri/jmrit/roster/swing/RosterTable.java or java/src/jmri/util/swing/multipane/TwoPaneTBWindow.java?

> Remove the manager variable, add the method
> 
> private GuiLafPreferencesManager manager() {
>     return InstanceManager.getDefault(GuiLafPreferencesManager.class);
> }
> And then replace all uses of manager with manager()